src/pathfinder/water_regions.cpp

```diff
@@ -49,7 +49,7 @@ class WaterRegionData {
 	std::array<TWaterRegionTraversabilityBits, DIAGDIR_END> edge_traversability_bits{};
 	std::unique_ptr<TWaterRegionPatchLabelArray> tile_patch_labels; // Tile patch labels, this may be nullptr in the following trivial cases: region is invalid, region is only land (0 patches), region is only water (1 patch)
 	bool has_cross_region_aqueducts = false;
-	TWaterRegionPatchLabel number_of_patches = 0; // 0 = no water, 1 = one single patch of water, etc...
+	int number_of_patches = 0; // 0 = no water, 1 = one single patch of water, etc...
```

Comment:
This unfortunately increased the water region size. And lately there have been a number of PRs to decrease the memory usage of water regions, so it would be a shame to increase it again. Just keep the TWaterRegionPatchLabel.

Reply:
:;)